src/Functional/TailRecursion.php

```diff
@@ -9,7 +9,7 @@
  * @param callable $fn
  * @return \Closure
  */
-function tailRec(callable $fn)
+function tailRecursion(callable $fn)
```

Comment:
Sorry, should be `tail_recursion` as we are not using camel case for function names